### ISSUE TITLE
Search not working 

### ISSUE THREAD

**Issue by rpanczer** [Microsoft/vscode#56607]
- VSCode Version: 1.26.0
- OS Version: macOS Mojave 10.14 Beta

Steps to Reproduce:

1. Search for anything in a project
2. Only results in .sql files appear

Does this issue occur when all extensions are disabled?: Yes

**Comment by roblourens** [comment]
Please see the wiki here for troubleshooting tips: https://github.com/Microsoft/vscode/wiki/Search-Issues

**Comment by rpanczer** [comment]
Thanks @roblourens. I tried setting `"search.useIgnoreFiles": false` and Use Ignore Flies to false but that did not work. I also noticed if I had a file open, the search would find the matching string in the open file, but if the file was closed the search returned no results.

**To fix it I set `Use Repgrep` to false.**  Not sure of the implications of this are but it works for now.

**Comment by roblourens** [comment]
Could you try this on Insiders? https://code.visualstudio.com/insiders/ And could you try something to help me figure out why ripgrep isn't working? 

- Run the command "Set log level" and change it to "Trace"
- Run the search again
- Show the search-related messages in the "Log (Window)" output panel

![image](https://user-images.githubusercontent.com/323878/35987940-02aae15a-0cb2-11e8-83f1-d95c5dc3881b.png)

**Comment by rpanczer** [comment]
Of course:
[output_ripgrep_search.txt](https://github.com/Microsoft/vscode/files/2302915/output_ripgrep_search.txt)

**Comment by roblourens** [comment]
So you're sure that the matches aren't in a file covered by the .gitignore? It looks like `"search.useIgnoreFiles": false` wasn't set in the search for that log.

Do you know what's different about .sql files that they do appear?

**Comment by rpanczer** [comment]
@roblourens Ah I see the problem. My fault.

I am using docker setup with submodules. The main module contains a .gitignore to ignore the submodule directories since they're checked in individually. When I checked the git ignore prior to submitting this ticket I looked in each submodule's .gitignore and confirmed .js was not ignored. I didn't think of checking the main .gitignore file, since I've been using this setup for a while now and never encountered this issue. The sql files were outside the submodule directories.

In any case thanks, for the help and patience.
